x-pack/plugins/cloud_security_posture/public/pages/vulnerabilities/vulnerabilities.tsx

```diff
@@ -4,17 +4,236 @@
+          !vulnerability.finding?.vulnerability.score.base ||
+          !vulnerability?.finding.vulnerability.severity
+        ) {
+          return null;
+        }
+        return (
+          <SeverityStatusBadge
+            score={vulnerability.finding.vulnerability.score.base}
+            status={vulnerability.finding.vulnerability.severity}
+          />
+        );
+      }
+      if (columnId === 'package-version') {
+        return (
+          <>
+            {vulnerability.finding?.vulnerability.package.name}{' '}
+            {vulnerability.finding?.vulnerability.package.version}
+          </>
+        );
+      }
+      if (columnId === 'fix-version') {
+        return (
+          <>
+            {vulnerability.finding?.vulnerability.package.name}{' '}
+            {vulnerability.finding?.vulnerability.package.fixed_version}
+          </>
+        );
+      }
+    };
+  }, [data?.page]);
+
+  const error = queryError || null;
+
+  if (error) {
+    return <ErrorCallout error={error as Error} />;
+  }
+  if (isLoading || !data) {
+    return <EuiLoadingSpinner />;
+  }
+
+  const columns = getVulnerabilitiesColumns();
+
   return (
     <>
-      <EuiSpacer size="l" />
-      <EuiCallOut
-        title="Placeholder for the Vulnerabilities Management integration content"
-        iconType="iInCircle"
+      <FindingsSearchBar
+        dataView={dataView}
+        setQuery={(newQuery) => {
+          setUrlQuery({ ...newQuery, pageIndex: 0 });
+        }}
+        loading={isLoading}
       />
+      <EuiSpacer size="l" />
+      {!isLoading && data.page.length === 0 ? (
+        <EuiEmptyPrompt
+          iconType="logoKibana"
+          title={
+            <h2>
+              <FormattedMessage
+                id="xpack.csp.findings.resourceFindings.noFindingsTitle"
+                defaultMessage="There are no Findings"
+              />
+            </h2>
+          }
+        />
+      ) : (
+        <EuiDataGrid
+          css={css`
+            & .euiDataGridHeaderCell__icon {
+              display: none;
+            }
+            & .euiDataGrid__controls {
+              border-bottom: none;
+            }
+            & .euiButtonIcon {
+              color: ${euiTheme.colors.primary};
+            }
+            & .euiDataGridRowCell {
+              font-size: ${euiTheme.size.m};
+            }
+          `}
+          aria-label="Data grid styling demo"
+          columns={columns}
+          columnVisibility={{
+            visibleColumns: columns.map(({ id }) => id),
+            setVisibleColumns: () => {},
+          }}
+          rowCount={data?.total}
+          toolbarVisibility={{
+            showColumnSelector: false,
+            showDisplaySelector: false,
+            showKeyboardShortcuts: false,
+            additionalControls: {
+              left: {
+                prepend: (
+                  <EuiButtonEmpty size="xs" color="text">
+                    {data?.total} Vulnerabilities
```

Comment:
Do we need to translate for the demo? @kfirpeled  @opauloh

Reply:
Good catch, we can translate this